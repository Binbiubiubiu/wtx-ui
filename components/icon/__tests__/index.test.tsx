import React from 'react';
import { render } from 'enzyme';
import ImageIcon from '..';
import toJson from 'enzyme-to-json';

describe('Icon', () => {
  it('should render Icon components', () => {
    const wrapper = render(
      <>
        <ImageIcon path={'test.png'} width="80" height="80" />
        <ImageIcon path={'test.png'} width="80" height="80" onClick={() => {}} />
      </>,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should style output is not null', () => {
    expect(require('../style')).not.toBeNull();
  });
});
