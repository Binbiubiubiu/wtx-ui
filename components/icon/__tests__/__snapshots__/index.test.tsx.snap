// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render Icon components 1`] = `
Array [
  <img
    alt=""
    class="wtx-image-icon"
    src="test.png"
    style="width:80;height:80"
  />,
  <img
    alt=""
    class="wtx-image-icon wtx-image-icon__clickable"
    src="test.png"
    style="width:80;height:80"
  />,
]
`;
